app/json/slate_lineup_configs.json

```diff
@@ -1,7 +1,7 @@
 [
   {
-    "id": "20481076-2261-4916-801a-fd186986367b",
-    "description": "Android Discover Tab Personalized Curated",
+    "id": "aaf47c0f-1361-4c8c-a89f-fa45e1dc2978",
```

Comment:
same concern as @jpetto here

Reply:
That ID was used in an old experiment, but just to be safe I re-added the old lineup id. It shouldn't be getting used but I am still seeing some analytics events for it 🙃